### PR TITLE
Bookstore: Use Redux in React components.

### PR DESCRIPTION
**Pull request by sanieni6**
# Bookstore: Use Redux in React components. 🤓 
In this step I used a Redux store to display books. I implemented react-redux in order to connect your store and then access your state variables.
## Summary: 📖 
+ Create an initialState variable for the booksSlice ✅ 
+ Display the books, received from the slice, in a reusable component ✅ 
+ Dispatch actions using useDispatch ✅ 
+ Add an event handler that adds a book to the books array ✅ 
+ Add an event handler that removes a book from the books array ✅

### CODE REVIEW

File: src/redux/books/booksSlice.js [hunk 1]
```diff
@@ -1,8 +1,28 @@
 /* eslint-disable no-param-reassign */
```
Comment (youmari) [inline src/redux/books/booksSlice.js]: - [x] it would be better to remove the eslint disable because it ensures that you write clean code

**Review by youmari — APPROVED** [review]
Hi @sanieni6  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Great commit messages
✔️ PRO PR
✔️ No linter error 

## Required Changes ♻️

- [x] it would be better when you click on the status button it should be dispatch an action to the redux store that will return a string "under construction"
Screenshot 2023-04-04 at 21 48 01

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/books/booksSlice.js (hunk 1 above)
Comment (sanieni6) [reply src/redux/books/booksSlice.js]: thanks for your review @youmari

**Comment by sanieni6** [comment]
# Changes done 🟡 
+ Add dispatch action checkStatus
+ Remove the disabled eslint

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @sanieni6 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**